dump1090.c

```diff
@@ -667,14 +667,14 @@ void showHelp(void) {
 "| dump1090 ModeS Receiver     %45s |\n"
 "-----------------------------------------------------------------------------\n"
 "--device-index <index>   Select RTL device (default: 0)\n"
-"--gain <db>              Set gain (default: max gain. Use -10 for auto-gain)\n"
```

Comment:
Oh, right. The option parsing scales it by a factor of 10 since rtlsdr wants an integer; so the original help was actually correct.